architecture/core_concepts/pods_and_services.adoc

```diff
@@ -507,6 +507,104 @@ incoming service connections between the endpoint pods. It also requires that
 all endpoints are always able to accept connections.
 endif::[]
 
+[[headless-services]]
+=== Headless services
+If your application does not need load balancing or single-service IP, you can create a headless service. When you create a headless service, no load-balancing or proxying is done and no cluster IP is allocated for this service. For such services DNS is automatically configured depending on whether the service has selectors defined or not.
+
+*Services with selectors*: For headless services that define selectors, the endpoints controller creates `Endpoints` records in the API, and modifies the DNS configuration to return `A` records (addresses) that point directly to the pods backing the service.
+
+*Services without selectors*: For headless services that do not define selectors, the endpoints controller does not create `Endpoints` records. However, the DNS system looks for and configures either:
+
+* `CNAME` records for `ExternalName` type services. Or,
+* `A` records for any endpoints that share a name with the service, for all other types.
+
+[[headless-service-creation]]
+==== Creating a headless service
+Creating a headless service is similar to the creation of a standard service, but it does not declare the `ClusterIP`. To create a headless service, add `clusterIP: None` in the service YAML definition.
```

Comment:
Maybe:
Creating a headless service is similar to creating a standard service, but you do not declare the `ClusterIP` address. To create a headless service, add the `clusterIP: None` parameter value to the service YAML definition.